src/pages/Missions.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import store from '../redux/configureStore';
+import {
+  getMissions,
+  removeReservation,
+  addReservation,
+} from '../redux/api/api';
+
+const Missions = (props) => {
+  const dispatch = useDispatch();
+  const [missionsDisplay, setMissionsDisplay] = useState(null);
+  const [calledMission, setCalledMission] = useState(null);
+
+  // eslint-disable-next-line no-unused-vars
+  const removeReserve = (e) => {
+    const { id } = e.currentTarget;
+    const data = { target: 'missions', id };
+    dispatch(removeReservation(data));
+  };
+
+  const reserve = (e) => {
+    const { id } = e.currentTarget;
+    const data = { target: 'missions', id };
+    dispatch(addReservation(data));
+  };
+
+  useEffect(() => {
+    const { missions } = props;
```

Comment:
- [ ] Also retrieve missions with the useSelector hook.

Reply:
Ok. thanks for pointing it out :+1: